adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -32,12 +31,111 @@ This extension adds new classes `command_graph` and `node` which follows the
 
 ==== Sub-Graph
 
-=== Node
 
-[source, c++]
+[[sec:khr-graph-node]]
```

Comment:
I'm not 100% on this, but looking at the main spec I feel like the `sec:` prefix is for top level sections, i.e `==` deep. `===` deep or more is `subsect`. 
```suggestion
[[subsec:khr-graph-node]]
```

Same comment for other occurrences

Reply:
I've bump all the section levels by one now.